parser/parse.go

```diff
@@ -749,3 +753,14 @@ func (b *Builder) addVariable(u types.Universe, useName *types.Name, in *tc.Var)
 	out.Underlying = b.walkType(u, nil, in.Type())
 	return out
 }
+
+// canonicalizeImportPath takes an import path and returns the actual package
+// it doesn't work for any folder actually called `vendor` (we don't dep any today),
```

Comment:
I would change this comment that it doesn';t support nested vendoring, but since we don't do that and we don't know anyone who does that it's a reasonable asumption.